src/plugins/intel_gna/CMakeLists.txt

```
@@ -6,8 +6,13 @@ if (NOT ENABLE_INTEL_GNA)
     return()
 endif()
 
+
 set(TARGET_NAME "openvino_intel_gna_plugin")
 
+if (ENABLE_INTEL_GNA_DEBUG)
+    add_definitions(-DGNA_DEBUG=1)
```

```suggestion
    add_compile_definitions(GNA_DEBUG)
```
command has been superseded by alternative